message.go

```diff
@@ -911,9 +925,11 @@ func (bs *BodyStructure) Parse(fields []interface{}) error {
 			if disp, ok := fields[end].([]interface{}); ok && len(disp) >= 2 {
 				if s, ok := disp[0].(string); ok {
 					bs.Disposition, _ = decodeHeader(s)
+					bs.Disposition = strings.ToLower(bs.Disposition)
 				}
 				if params, ok := disp[1].([]interface{}); ok {
 					bs.DispositionParams, _ = parseHeaderParamList(params)
+					keysLowered(bs.DispositionParams)
```

Comment:
I think we can inline the lower-casing in `parseHeaderParamList`.

Reply:
Can we do that in `ParseParamList`? I think that's the most clean as the map is first populated there, but I didn't want to mess with a public/exported function.

Reply:
Yeah, we can't do that because that would be a breaking change.